packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -148,26 +149,48 @@ const extractAttachmentUrl = (attachment: ChatAttachment): string => {
   return attachment.url || '';
 };
 const processChatMessageContent = (message: ChatMessageWithStatus): string | undefined => {
+  let content = message.content?.message;
   /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
-  if (sanitizedMessageContentType(message.type).includes('html') && message.content?.attachments) {
+  if (content && message.content?.attachments && sanitizedMessageContentType(message.type).includes('html')) {
     const attachments: ChatAttachment[] = message.content?.attachments;
+    // Fill in the src here
+    /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
```

Comment:
Do we need `@conditional-compile-remove` inside of `if` statement as everything in `if` should be removed?

Reply:
Fixed